app/assets/stylesheets/pages/_all.scss

```diff
@@ -0,0 +1,3 @@
+@import "home";
+@import "machines";
+@import "roasts";
```

Comment:
This is the best directory structure. Keeps us from referencing everything into our `main.scss`.